src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Windows.cs

```diff
@@ -710,9 +712,29 @@ public static SocketError Poll(SafeCloseSocket handle, int microseconds, SelectM
 
         public static SocketError Select(IList checkRead, IList checkWrite, IList checkError, int microseconds)
         {
-            IntPtr[] readfileDescriptorSet = Socket.SocketListToFileDescriptorSet(checkRead);
-            IntPtr[] writefileDescriptorSet = Socket.SocketListToFileDescriptorSet(checkWrite);
-            IntPtr[] errfileDescriptorSet = Socket.SocketListToFileDescriptorSet(checkError);
+            bool ShouldStackAlloc(IList list, out Span<IntPtr> span)
```

Comment:
@stephentoub i found [this](https://github.com/dotnet/corefx/pull/29617#issuecomment-387881742) on issues but nothing on [coding style guide](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), is there a guideline?

Reply:
Local functions are fine style-wise.